spec/smoke_test/job_seekers_can_view_homepage_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'browser_test_helper'
+require 'pry'
+
+RSpec.describe 'Page availability', js: true do
+  context 'Job seeker visits home page' do
+    it 'Page should be available' do
+      page = Capybara::Session.new(:poltergeist)
+      page.visit 'https://teaching-vacancies.service.gov.uk/'
+      expect(page).to have_content('Find a job in teaching')
+    end
+
+    it 'Job search should return a result' do
```

Comment:
Can we merge this into the above `it` assertion block so that we have a single test for the user journey being tested? That way, we only visit the homepage the one, assert on that, then move onto the next page etc? If at any point the assertion fails, the test will too.

Reply:
Yes that makes sense, resolved this in cd60e72